README.md

```diff
@@ -23,6 +23,11 @@ You can find the most recent version of this guide [here](https://github.com/fac
    `npm run start:mock-instrospect-schema` will obtain a GQL schema via intropspection from http://localhost:9090/graphql/query and mock GQL responses
    `npm run start:mock-custom-schema` will obtain the schema from .build-dev-cmdrc.json and mock GQL responses
 
+### GQL Query Linting
+
+Before running locally, symlink your GraphQL schema in formatted in the schema definition language to a file named sdlschema in the root of the Spruce directory to enable query linting with ESlint like so `ln -s /path/to/schema sdlschema`
+Run `npm run eslint` to see the results of query linting in your terminal or install a plugin to integrate ESlint into your editor.
+
```

Comment:
+1

Reply:
Had to read this a few times and still don't quite understand "symlink your GraphQL schema in formatted in the schema definition language"

Reply:
Maybe bullets would help break up this run-on sentence?

Reply:
I would mention here what vscode plugin to install

Reply:
based off these instructions i'm still not totally sure how to fetch my own schema. explicit step by step instructions could help?